README.md

```diff
@@ -208,6 +208,17 @@ arecord -f S16_LE -c1 -r 16000 -t raw -D default | nc localhost 43001
 
 - nc is netcat with server's host and port
 
+#### Client example on the Mac system:
+
+List microphones:
+```
+ffmpeg -hide_banner -f avfoundation -list_devices true -i ""
+```
+
+Live stream raw audio:
+```
+ffmpeg -hide_banner -f avfoundation -i ":0" -ac 1 -ar 48000 -acodec pcm_s16le -ar 16000 -f s16le -loglevel error - | nc localhost 50000
```

Comment:
Thanks.

`-ar 48000` seems to be redundant, because there is `-ar 16000`. 

And `-acodec pcm_s16le` and `-f s16le` seem to be duplicated.

Can you check if these options could be simpler? 

Also, can you expand what to do with list of microphones? You probably need to pass the `:0` from the list to the other command, right? Make it explicit.